.github/workflows/kicad-production.yml

```
@@ -0,0 +1,326 @@
+        echo "3. Reference the schematic PDF for testing and troubleshooting" >> $GITHUB_STEP_SUMMARY
+        echo "4. Import the STEP file for mechanical design verification" >> $GITHUB_STEP_SUMMARY
+        
+    - name: Comment on PR with Artifact Links
+      if: github.event_name == 'pull_request'
+      uses: actions/github-script@v7
+      with:
+        script: |
+          const runId = context.runId;
+          const repoUrl = `${context.serverUrl}/${context.repo.owner}/${context.repo.repo}`;
+          const artifactUrl = `${repoUrl}/actions/runs/${runId}`;
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)